notebooks/113-image-classification-quantization/113-image-classification-quantization.ipynb

```diff
@@ -0,0 +1,446 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "rQc-wXjqrEuR"
+      },
+      "source": [
+        "# Quantization of Image Classification Models\n",
+        "\n",
+        "This tutorial demostrates how to apply INT8 quantization to Image Classification model using [Post-training Optimization Tool API](../../compression/api/README.md). The Mobilenet V2 model trained on Cifar10 dataset is used as an example. The code of this tutorial is designed to be extandable to custom model and dataset. It consists of the following steps:\n",
+        "- Install OpenVINO and required tools and packages using PIP manager\n",
```

Comment:
The OpenVINO installation was previously performed before opening this notebook. Therefore, I suggest removing this line and mentioning that the libraries are imported and the tools used.